src/main/java/gan/missulgan/member/controller/MemberController.java

```diff
@@ -20,6 +21,19 @@ public class MemberController {
 	@GetMapping("me")
 	@ApiOperation(value = "유저 정보 가져오기", notes = "현재 인증된 유저 정보 가져옴")
 	public MemberDTO getMember(@AuthenticatedEmail String email) {
-		return memberService.findUser(email);
+		return memberService.findMember(email);
+	}
+
+	@GetMapping("nickname")
+	@ApiOperation(value = "별명 변경 전 카카오톡 이름 보여주기")
+	public UserNicknameResponse getUserNickname(@AuthenticatedEmail String email) {
+		return new UserNicknameResponse(memberService.findMember(email).getUserNickname());
 	}
+
+	@PutMapping("nickname")
+	@ApiOperation(value = "별명 변경하기")
+	public UserNicknameResponse putUserNickname(@RequestBody UserNicknameRequest userNicknameRequest) {
+		return new UserNicknameResponse(memberService.saveUserNickname(userNicknameRequest));
+	}
+
```

Comment:
이 부분에서 사용자를 구분할 email을 DTO로 직접받는것 같은데 그럴 필요 없어요.
위에서처럼 @AuthenticatedEmail로 해결이 가능합니다. DTO에서는 accountEmail 제거 부탁드릴게요.
지금 상태로는 회원이기만 하면 누구나 남의 별명을 수정할 수 있어요.
```suggestion

	@PutMapping("nickname")
	@ApiOperation(value = "별명 변경하기")
	public UserNicknameResponse putUserNickname(@AuthenticatedEmail String email, @RequestBody UserNicknameRequest userNicknameRequest) {
		return new UserNicknameResponse(memberService.saveUserNickname(userNicknameRequest));
	}

```